prow/README.md

```diff
@@ -1,243 +1,75 @@
-```
-
-Batch Job:
-```json
-{"type":"batch","job":"job-name","buildid":"0","prowjobid":"uuid","refs":{"org":"org-name","repo":"repo-name","base_ref":"base-ref","base_sha":"base-sha","pulls":[{"number":1,"author":"author-name","sha":"pull-sha"},{"number":2,"author":"other-author-name","sha":"second-pull-sha"}]}}
-```
-
-## Bots home
-
-[@k8s-ci-robot](https://github.com/k8s-ci-robot) and its silent counterpart
-[@k8s-bot](https://github.com/k8s-bot) both live here as triggers to GitHub
-messages defined in [config.yaml](config.yaml). Here is a
-[command list](https://go.k8s.io/bot-commands)
-for them.
+For a brief overview of how Prow runs jobs take a look at ["Life of a Prow Job"](/prow/life_of_a_prow_job.md).
+
+#### Functions and Features
+* Job execution for testing, batch processing, artifact publishing…
+    * GitHub events are used to trigger post-PR-merge (postsubmit) jobs and on-PR-update (presubmit) jobs.
+    * Support for multiple execution platforms and source code review sites.
+* Pluggable GitHub bot automation that implements ‘slash’ commands and enforces configured policies/processes.
+* GitHub merge automation with batch testing logic.
+* Front end for viewing jobs, merge queue status, dynamically generated help information, and more.
+* Automatic deployment of source control based config.
+* Automatic GitHub org/repo administration configured in source control.
+
+* Designed for multi-org scale with dozens of repositories. (The Kubernetes Prow instance uses only 1 GitHub bot token!)
+* High availability as benefit of running on Kubernetes. (replication, load balancing, rolling updates...)
+* JSON structured logs.
+* Prometheus metrics.
+
+
+## Documentation
+
+### Getting started...
+
+* ... with your own Prow deployment: [`getting_started_deploy.md`](/prow/getting_started_deploy.md)
+* ... with developing for Prow: [`getting_started_develop.md`](/prow/getting_started_develop.md)
+* ... as a job author: [`jobs.md`](/prow/jobs.md)
+
+### More details
+- [Components](/prow/cmd/README.md)
+- [Plugins](/prow/plugins/README.md)
+- [ProwJobs](/prow/jobs.md)
+- [Testing and Debugging](/prow/test_debug.md)
+- [General Configuration](/prow/config/README.md)
+- [Pod Utilities](/prow/pod-utilities.md)
+- [Scaling Prow](/prow/scaling.md)
+- [Tide](/prow/cmd/tide/README.md)
+- [Metrics](/prow/metrics/README.md)
+- ["Life of a Prow Job"](/prow/life_of_a_prow_job.md)
+- [Getting more our of Prow](/prow/more_prow.md)
 
 ## Useful Talks
 
 ### KubeCon 2018 EU
 [Automation and the Kubernetes Contributor Experience](https://www.youtube.com/watch?v=BsIC7gPkH5M)
 
 [SIG Testing Deep Dive](https://www.youtube.com/watch?v=M32NIHRKaOI)
+
+## Prow in the wild
+
+We know that Prow is used by the following organizations:
+- [Kubernetes](https://prow.k8s.io)
+- [OpenShift](https://deck-ci.svc.ci.openshift.org/)
+- [Istio](https://prow.istio.io/)
+- [Knative](https://prow.knative.dev/)
+- Jetstack
```

Comment:
https://prow.build-infra.jetstack.net/

Reply:
cc @munnerz :-)

Reply:
https://github.com/jenkins-x/prow? 🙃

Reply:
I don't think there's a public deployment there for consistency with the others. There are definitely other deployments without accessible `deck`s